Update software third party controllers (#1140)
astronomer/docs: software/third-party-ingress-controllers.md

    tag: stable
```

> If you use a Nginx, Traefik or Contour ingress controller, you need to configure additional values in your chart. For more information, read the following subsections.

Reviewer: ```suggestion
If you use an Nginx, Traefik or Contour, ingress controller, you need to configure additional values in your chart. For more information, read the following subsections.
```

> ### Required configuration for ingress-nginx

Reviewer: ```suggestion
### Required configuration for nginx
```

> If you're using a nginx ingress controller, add the following configuration to your `config.yaml` file:

Reviewer: ```suggestion
If use an Nginx ingress controller, add the following configuration to your `config.yaml` file:
```

```yaml
global:
  extraAnnotations:
    nginx.ingress.kubernetes.io/proxy-body-size: 0
```

> This setting will disable Nginx's maximum allowed size upload and prevent HTTP 413 (Request Entity Too Large) errors. 
> It is required for Astro CLI to properly deploy DAGs to the internal registry.

Reviewer: ```suggestion
This setting disables Nginx's maximum allowed upload size, which prevents HTTP 413 (Request Entity Too Large) errors. It also allows the Astro CLI to properly deploy DAGs to Astronomer Software's internal registry.
```

Reviewer: Are the Astro CLI behavior and the HTTP 413 error related? If so, I would instead change this to:

"This setting disables Nginx's maximum allowed upload size, which prevents HTTP 413 (Request Entity Too Large) error and allows the Astro CLI to properly deploy DAGs to Astronomer Software's internal registry."
